imports/math/shared.lua

```diff
@@ -90,4 +90,14 @@ function math.tohex(n, upper)
     return upper and n:upper() or n
 end
 
-return lib.math
+---Converts input number into grouped digits
+---@param number number
+---@param useDot? boolean
+---@return string
+function math.groupdigits(number, useDot) -- credit http://richard.warburton.it
+    local divider = useDot and '.' or ','
+    local left,num,right = string.match(number,'^([^%d]*%d)(%d*)(.-)$')
+    return left..(num:reverse():gsub('(%d%d%d)','%1' .. divider):reverse())..right
```

Comment:
might as well be `divider` or `seperator` as the arg (not useDot), and then just do `divider or ','`.